From 5cc1ad47793a85663740b77ee3eb1aa412a00739 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Tue, 30 Oct 2012 17:54:26 +0800
Subject: [PATCH 090/602] ENGR00229188 imx6 sabresd: keep the w/h size ratio
 for thumbnail

Keep the width/height ratio for activity screenshot thumbnail.
The original height of the thumbnail is 135dp, which
cause am use the height low ratio to get screenshot with
size of 199x144. This screenshot is too small to fit
the status bar's recent activity panel view.
Increase the height size for thumbnail by size of
225x170, which fit the image view and avoid blank
area on the right of the app thumbnail.

Change-Id: I96aed74e9a7af1538392a32daf16ec1c88764a98
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 .../base/core/res/res/values-sw720dp/dimens.xml    | 28 ++++++++++++++++++++++
 1 file changed, 28 insertions(+)
 create mode 100644 imx6/sabresd/overlay/frameworks/base/core/res/res/values-sw720dp/dimens.xml

diff --git a/imx6/sabresd/overlay/frameworks/base/core/res/res/values-sw720dp/dimens.xml b/imx6/sabresd/overlay/frameworks/base/core/res/res/values-sw720dp/dimens.xml
new file mode 100644
index 0000000..2418175
--- /dev/null
+++ b/imx6/sabresd/overlay/frameworks/base/core/res/res/values-sw720dp/dimens.xml
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+**
+** Copyright 2012, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources>
+
+    <!-- The width that is used when creating thumbnails of applications. -->
+    <dimen name="thumbnail_width">230dp</dimen>
+    <!-- The height that is used when creating thumbnails of applications. -->
+    <dimen name="thumbnail_height">170dp</dimen>
+
+</resources>
+
+
-- 
1.8.0

